src/mcts/params.cc

```diff
@@ -131,6 +140,11 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<IntOption>(kMaxCollisionVisitsId, 1, 1000000) = 1;
   options->Add<BoolOption>(kOutOfOrderEvalId) = false;
   options->Add<IntOption>(kMultiPvId, 1, 500) = 1;
+  std::vector<std::string> history_fill_opt;
+  history_fill_opt.push_back("no");
+  history_fill_opt.push_back("fen_only");
+  history_fill_opt.push_back("always");
+  options->Add<ChoiceOption>(kHistoryFillId, history_fill_opt) = "fen_only";
```

Comment:
I think `{"no", "fen_only", "always"}` will work without need of a temporary vector.

Reply:
I get errors with that, made it
```
  std::vector<std::string> history_fill_opt {"no", "fen_only", "always"};
  options->Add<ChoiceOption>(kHistoryFillId, history_fill_opt) = "fen_only";
```

Reply:
I think that's the right thing to do. In absolute terms, a temporary vector is necessary simply because ChoiceOption's constructor requires one. In a perfect world, you wouldn't need to spell it out (the vector could be implicitly created from an initializer_list), but that doesn't work with std::make_unique (a known limitation).

Reply:
Ah, uniform initialization can interfere with `Add()`'s perfect forwarding, but for initializer list it should work..
Anyway, it's fine as it is.